### DISCUSSION TITLE
Fixing Class "Database\Seeders\Csv\CsvToArray" not found

### DISCUSSION THREAD

**Discussion by Naoe-Adrian** [PUPT-FLSS-ORG/PUPT-FLSS#87] [Ideas]
**Fix for Seeder CSV Import 🚀**

Hi @Rasquero-ViaClariz  @Malaluan-KylaRica  kindly check this out 

![image](https://github.com/user-attachments/assets/deb414b4-be4a-4e35-ae7b-c235fa5373c6)

In relation to PR #86, the CsvToArray class now relies on the league/csv package for CSV import. To ensure the seeder works correctly, please run composer install to install the required dependencies:

**run: `php artisan migrate:refresh` (optional to avoid some db bugs and to clean your db)
run: `composer install`**

![image](https://github.com/user-attachments/assets/c5782fe4-d602-4bd3-8f34-cb8b85afe50b)

_This process will take some time and wait it dont close your terminal or your vscode_

**Then run: `php artisan migrate:fresh --seed`**

![image](https://github.com/user-attachments/assets/0f035794-b1f4-4b4b-9952-5dd6070f2d46)

![image](https://github.com/user-attachments/assets/978a9d3c-3256-4718-b11b-fc85e8e25320)

**Comment by Naoe-Adrian** [comment]
Let me know if this work around works on your local machine and if theres any other bugs you encounter on our latest pr alert me in our gc thank you

**Reply by Rasquero-ViaClariz** [reply]
Thank you for the update! I’ll follow the steps and test this on my local machine. I’ll let you know if I encounter any issues or bugs. Thanks for the detailed instructions!
